src/engine/ruleset/integrations/windows/decoders/windows-powershell.yml

```diff
@@ -71,14 +70,26 @@ normalize:
    - file.extension: $_file_data.ext
    - file.name: $_file_data.name
 
- - check: $event.code=="400"
+ - check: $event.code == '400'
    map:
     - event.type: array_append(start)
 
- - check: $event.code=="403"
+ - check: $event.code == '403'
    map:
     - event.type: array_append(end)
 
-# Cleanup
+# windows.logon
  - map:
-   - windows: delete()
+   - windows.logon.activity_id: $windows.System.Task.@ActivityID
+   - windows.logon.channel: $windows.System.Channel.#text
+   - windows.logon.computer_name: $windows.System.Computer.#text
+   - windows.logon.event_id: $windows.System.EventID.#text
+   - windows.logon.keywords: kvdb_decode_bitmask(windows_bitmask_tables, Keywords, $windows.System.Keywords.#text)
```

Comment:
Should  check the special case wjhens Keywords, `$windows.System.Keywords.#text == 0x0`